frontend/components/welcomeHeader.jsx

```diff
@@ -17,31 +18,64 @@ const TitleContainer = styled.div`
 `;
 
 const TimerContainer = styled.div`
-  align-self: flex-end;
+`;
+
+const InfoContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+`;
+
+const TableContainer = styled.div`
+  font-family: 'Madina Script';
+  src: url('MadinaScript.ttf')
+`;
+
+const SolvedContainer = styled.div`
 `;
 
 class WelcomeHeader extends React.Component {
-  render() {
-    const { gameState } = this.props;
+  constructor(props) {
+    super(props);
+    this.numSolved = this.numSolved.bind(this);
+  }
 
-    const { start_time } = gameState || {};
+  numSolved(gameState) {
+    var solvedCounter = 0;
+    if (gameState) {
+      return Object.values(gameState).reduce((solved, info) => {
+        return info['solved'] ? solved + 1 : solved;
+      }, 0);
+    };
+    return solvedCounter;
+  }
 
+  render() {
+    const { gameState, cookies } = this.props;
+    const { start_time } = gameState || {};
+    const table = cookies.get('table')
+    var solved = this.numSolved(gameState);
     return (
       <Header>
         <TitleContainer>
           <Title>Chois Escape</Title>
+        </TitleContainer>
+        <InfoContainer>
+          <TableContainer>Table {table}</TableContainer>
```

Comment:
i think it makes sense to only have the number itself be in the Madina font? what do you think?